docs/topics/dokkatoo-migration.md

```diff
@@ -0,0 +1,359 @@
+                  remoteLineSuffix.set("#L")
+              }
+          }
+      }
+  }
+
+  tasks.dokkaHtml {
+      pluginConfiguration<DokkaBase, DokkaBaseConfiguration> {
+          customStyleSheets.set(listOf("styles.css"))
+          customAssets.set(listOf("logo.png"))
+          footerMessage.set("(c) Your Company")
+      }
+  }
+  ```
+ 
+  New configuration:
+
+  ```kotlin
+  dokka {
+      moduleName = "Project Name"
+      dokkaSourceSets.main {
+          includes.from("README.md")
+          sourceLink {
+              localDirectory = file("src/main/kotlin")
+              remoteUrl = URL("https://example.com/src")
+              remoteLineSuffix = "#L"
+          }
+      }
+      pluginsConfiguration.html {
+          customStyleSheets.from("styles.css")
+          customAssets.from("logo.png")
+          footerMessage = "(c) Your Company"
+      }
+  }
+  ```
+
+* **Visibility settings:**  The `documentedVisibilities` property has changed from `Visibility.PUBLIC` to `VisibilityModifier.Public`.
+
+  Previous configuration:
+
+  ```kotlin
+  import org.jetbrains.dokka.DokkaConfiguration.Visibility
+  ...
+  documentedVisibilities.set(
+      setOf(Visibility.PUBLIC)
+  )  
+  ```
+
+  New configuration:
+
+  ```kotlin
+  import org.jetbrains.dokka.gradle.engine.parameters.VisibilityModifier
+  ...
+  documentedVisibilities.set(
+      setOf(VisibilityModifier.Public)
+  )
+  ```
+
+* **External documentation link:** The source link configuration has changed to accommodate Java and Gradle deprecations. 
+  The remote URL is now specified using `URI` instead of `URL`.
+
+  Previous configuration:
+  
+    ```kotlin
+    remoteUrl.set(URL("https://github.com/your-repo"))   
+    ```
+  
+  New configuration:
+  
+    ```kotlin
+    remoteUrl.set(URI("https://github.com/your-repo"))  
+    ```
+
+* **Custom assets:** The [`customAssets`](dokka-html.md#customize-assets) property now uses collections of files (`FileCollection`) instead of lists (`var List<File>`).
+
+  Previous configuration:
+
+    ```kotlin
+    customAssets = listOf(file("example.png"), file("example2.png"))   
+    ```
+
+  New configuration:
+
+    ```kotlin
+    customAssets.from("example.png", "example2.png")
+    ```
+
+### Share Dokka configuration across modules
+
+DPG v2 moves away from using `subprojects {}` or `allprojects {}` to share configuration across modules. In future Gradle versions, 
+using these approaches will lead to errors. For more information, see [Gradle's documentation about isolated projects](https://docs.gradle.org/current/userguide/isolated_projects.html).
+
+Follow the steps below to properly share Dokka configuration in multi-module projects with DPG v2.
+
+> For a multi-module project example, see the [Dokka GitHub repository](https://github.com/Kotlin/dokka/tree/master/examples/gradle/dokka-multimodule-example).
+>
+{type="tip"}
+
+#### Multi-module projects without convention plugins
```

Comment:
I would still link here to a convention plugin like we do in the other branch of this section.

Reply:
Linked from a sentence a bit below to avoid linking from the heading.